src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/commentsindentation/InputCommentsIndentationCheckWithEmoji.java

```diff
@@ -0,0 +1,112 @@
+package com.puppycrawl.tools.checkstyle.checks.indentation.commentsindentation;
+
+public class InputCommentsIndentationCheckWithEmoji {
+
+    public void myMethod() {
+        String breaks = "J"
+        // warn 'Indentation should be the same level as line 8'
```

Comment:
Please add some emoji to comment contents throughout file.

Reply:
done

Reply:
Resolved.